recipes/leftbar/general_row/general_row.vue

```diff
@@ -45,11 +47,13 @@ import { LEFTBAR_GENERAL_ROW_TYPES, LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS }
 import { DtBadge } from '@/components/badge';
 import DtEmojiTextWrapper from '@/components/emoji_text_wrapper/emoji_text_wrapper.vue';
 import DtRecipeLeftbarGeneralRowIcon from './leftbar_general_row_icon.vue';
+import DtButton from '@/components/button/button.vue';
```

Comment:
Independent of `d-btn` style concerns I mentioned in previous comment, is there functionality that using `<dt-button>` provides?

Reply:
👍 Yep, rip it out.